vite.config.ts

```diff
@@ -24,21 +23,20 @@ export default defineConfig({
 
     rollupOptions: {
       // Must be in sync with the list of dependencies in the package.json
-      // It includes all dependencies except @vector-im/compound-design-tokens
       external: [
         "react",
         "react/jsx-runtime",
         "react/jsx-dev-runtime",
         "classnames",
         "vaul",
         "@floating-ui/react",
-        "@floating-ui/react-dom",
         "@radix-ui/react-context-menu",
         "@radix-ui/react-dropdown-menu",
         "@radix-ui/react-form",
         "@radix-ui/react-separator",
         "@radix-ui/react-slot",
         "@radix-ui/react-tooltip",
+        /^@vector-im\/compound-design-tokens\/.*/,
```

Comment:
Can you comment on why this isn't just `"@vector-im/compound-design-tokens"`?

Reply:
I added a comment, hopefully clear enough?